conan/tools/microsoft/msbuilddeps.py

```diff
@@ -12,6 +12,13 @@
 VALID_LIB_EXTENSIONS = (".so", ".lib", ".a", ".dylib", ".bc")
 
 
+def msbuild_arch(arch):
+    return {'x86': 'x86',
```

Comment:
```suggestion
    return {'x86': 'Win32',
```

I think this is the right one, and the reason the unit test is failing

Reply:
:facepalm:

Reply:
You also need to check the CLA signing above, please